tests/acceptance/test_organization_integration_external_mappings.py

```diff
@@ -24,6 +24,7 @@ def setUp(self):
             name="getsentry/sentry",
             provider="integrations:github",
             integration_id=self.integration.id,
+            project=self.project,
```

Comment:
needed for the repository to actually be tied to an organization (the factory does `organization_id=project.organization_id`)